README.md

```diff
@@ -4,207 +4,20 @@
-
-```sh
-python3 -m pip install --upgrade pip setuptools wheel
-```
-
-To install this package from source, clone this repository and run the following command from the project root:
-
-```sh
-python3 -m pip install .
-```
-
-### Run the samples
-
-See the [examples](./examples) directory for a list of sample orchestrations and instructions on how to run them.
-
-## Development
-
-The following is more information about how to develop this project. Note that development commands require that `make` is installed on your local machine. If you're using Windows, you can install `make` using [Chocolatey](https://chocolatey.org/) or use WSL.
-
-### Generating protobufs
-
-```sh
-pip3 install -r dev-requirements.txt
-make gen-proto
-```
-
-This will download the `orchestrator_service.proto` from the `microsoft/durabletask-protobuf` repo and compile it using `grpcio-tools`. The version of the source proto file that was downloaded can be found in the file `durabletask/internal/PROTO_SOURCE_COMMIT_HASH`.
-
-### Running unit tests
-
-Unit tests can be run using the following command from the project root. Unit tests _don't_ require a sidecar process to be running.
-
-```sh
-make test-unit
-```
-
-### Running E2E tests
-
-The E2E (end-to-end) tests require a sidecar process to be running. You can use the Dapr sidecar for this or run a Durable Task test sidecar using the following `docker` command:
-
-```sh
-docker run --name durabletask-sidecar -p 4001:4001 --env 'DURABLETASK_SIDECAR_LOGLEVEL=Debug' --rm cgillum/durabletask-sidecar:latest start --backend Emulator
-```
-
-To run the E2E tests, run the following command from the project root:
-
-```sh
-make test-e2e
-```
-
-## Contributing
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# References
+- [Supported Patterns](./docs/supported-patterns.md)
+- [Available Features](./docs/features.md)
+- [Getting Started](./docs/getting-started.md)
+- [Development Guide](./docs/development.md) 
+- [Contributing Guide](./docs/development.md)
 
 ## Trademarks
```

Comment:
nit: I see you're removing whitespace between titles and content, but MS Learn markdown conventions require an extra newline between section titles and content. I suggest we do the same here.

Suggested VS Code extension: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint